.github/workflows/build-android.yml

```diff
@@ -83,18 +83,19 @@ jobs:
   build-android:
     needs: setup
     name: Build Android
-    runs-on: macOS-15
+    runs-on: macOS-latest
     steps:
       - name: Disk Cleanup
         run: |
           echo "::group::Free space before cleanup"
           df -hI
           echo "::endgroup::"
           echo "::group::Cleaned Files"
-          sudo rm -rf /Applications/Xcode_15.4.app
-          sudo rm -rf /Applications/Xcode_16.1.app
-          sudo rm -rf /Applications/Xcode_16.2.app
-          sudo rm -rf /Applications/Xcode_16.3_Release_Candidate_2.app
+          sudo rm -rf /Applications/Xcode_14.3.1.app
+          sudo rm -rf /Applications/Xcode_15.0.1.app
+          sudo rm -rf /Applications/Xcode_15.1.app
+          sudo rm -rf /Applications/Xcode_15.2.app
+          sudo rm -rf /Applications/Xcode_15.3.app
```

Comment:
Could you revert these changes?

Reply:
Done and re-triggered builds

Reply:
This affects more moving parts than I initially assumed. It's okay to revert back to macOS-latest runner for Android build.